quickwit/quickwit-serve/src/jaeger_api/model.rs

```diff
@@ -57,6 +57,7 @@ pub struct TracesSearchQueryParams {
     pub operation: Option<String>,
     pub start: Option<i64>,
     pub end: Option<i64>,
+    pub tags: Option<String>,
```

Comment:
not sure if this is the best way, maybe a `HashMap<String, serde_json:Value>` could help